logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -302,6 +302,23 @@ sealed class Event(open val id: String, open val transient: Boolean) {
                 senderUserIdKey to senderUserId.toLogString(),
             )
         }
+
+        data class ConversationMessageTimer(
```

Comment:
I think messageTimer does not explain the purpose of this well enough, for someone outside the context